browser/js/adminDash/users/adminDash.userForm.html

```diff
@@ -0,0 +1,17 @@
+<form class="admin-edit-form" ng-submit="updateOrCreate(user)">
+  <label ng-hide="newSong">ID</label>
+  <input ng-hide="newSong" type="text" ng-model="user.id" ng-disabled="true">
+  <h3 ng-show="newSong">Create a New Song</h3>
+  <label>First Name</label>
```

Comment:
I think the ng-hide/ng-shows should be for newUser?  And the h3 should say Create a New User